test/test.js

```diff
@@ -42,8 +42,11 @@ function joinAndForward(d) {
 
 const list = [];
 
+console.log('FLAVOR = ', flavor);
 if (flavor === 'only-npm') {
   list.push(joinAndForward('test-79-npm/main.js'));
+  list.push(joinAndForward('test-1191/main.js'));
```

Comment:
IMO it is cleaner to move these two tests to `test-79-npm` folder, so we can remove the special logics here (and several lines below).

Maybe `better-sqlite3` and `express-route` folders?

Reply:
I've added a specific test for better-slite in the test-79 series

however, I kept test test-1191  & test-1992  as they are testing different things.
I simply enabled them to run when platform === "host" as the incorporated binary module would be mismatching the targeted node version.